src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSourceContext.java

```diff
@@ -86,6 +86,11 @@
      */
     private boolean notificationsDisabled;
 
+    /**
+     * {@code true} if unstable builds should be considered as successful by Bitbucket.
+     */
+    private boolean isUnstableBuildSuccess;
```

Comment:
The name should make it clear that it is setting the notification sent, not changing the actual result of the run.  I'm not sure of the best way to say it. 

```suggestion
    private boolean notifyUnstableBuildAsSuccessful;
```

Or
```suggestion
    private boolean notifySuccessfulForUnstableBuild;
```

Or 
```suggestion
    private boolean sendSuccessNotificationForUnstableBuild
```

Reply:
I have renamed the method to be more descriptive.